### PR TITLE
(DOCSP-39165) Node and RN: Change baseURL during runtime

### PR DESCRIPTION
**Pull request by krollins-mdb**
## Pull Request Info

Jira ticket: https://jira.mongodb.org/browse/DOCSP-39165

- [Node.js: Connect to an Atlas App Services Backend](https://preview-mongodbkrollinsmdb.gatsbyjs.io/realm/DOCSP-39165/sdk/node/app-services/connect-to-app-services-backend/#connect-to-a-specific-server)
- [React Native: Connect to an Atlas App Services App](https://preview-mongodbkrollinsmdb.gatsbyjs.io/realm/DOCSP-39165/sdk/react-native/app-services/connect-to-app-services-app/#connect-to-a-specific-server)

This PR adds content to both the Node.js and React Native SDKs for changing the base URL during runtime. However, it only includes examples for the Node.js SDK. Right now, the experimental module does not really work for React Native. So, the React Native content notes that this feature exists, but is experimental and points to the Node.js content.

Sorry for the large number of changed files. I couldn't resist removing an old React Native app that we weren't using for anything.

Build errors:
-  There's a build error about not being able to find `label:node-change-server-during-runtime`. Manually testing shows the ref link working.
- Looks like there are a couple flaky React Native tests. They're unrelated to any changes I've made. I'll need to set aside time to look at them later.

### Reminder Checklist

Before merging your PR, make sure to check a few things.

- [x] Did you tag pages appropriately?
  - genre
  - meta.keywords
  - meta.description
- [x] Describe your PR's changes in the Release Notes section
- [x] Create a Jira ticket for related docs-app-services work, if any

### Release Notes

- **Node.js SDK**
  - App Services/Connect to an Atlas App Services Backend: Add new sections and examples for updating the baseUrl, including during runtime.
- **React Native SDK**
  - App Services/Connect to an Atlas App Services App: Add new sections that address updating the baseUrl.

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-realm/blob/master/REVIEWING.md)

### CODE REVIEW

File: source/sdk/react-native/app-services/connect-to-app-services-app.txt [hunk 1]
```diff
@@ -93,3 +93,32 @@ To encrypt App metadata:
 #. Pass the App configuration object to ``new Realm.App()``.
 
 .. include:: /examples/generated/react-native/v12/EncryptMetadata.snippet.encrypt-metadata.tsx.rst
+
+Connect to a Specific Server
```
Comment (lindseymoore) [inline source/sdk/react-native/app-services/connect-to-app-services-app.txt:97]: Is it not possible to include a code example for passing in a baseURL? Would be nice to include an example if possible.

**Review by lindseymoore — CHANGES_REQUESTED** [review]
A few nits and a question about including code examples for passing in a baseURL for RN.

File: source/sdk/react-native/app-services/connect-to-app-services-app.txt (hunk 1 above)
Comment (krollins-mdb) [reply source/sdk/react-native/app-services/connect-to-app-services-app.txt:97]: Not a tested code example. It simply doesn't work right now with TypeScript.

File: source/sdk/react-native/app-services/connect-to-app-services-app.txt [hunk 2]
```diff
@@ -93,3 +93,32 @@ To encrypt App metadata:
...
+.. versionchanged:: ``realm@12.8.0``
+
+In some cases, you might want to change the ``baseURL`` while the app is
+running. For example, you might want to roam between Edge Servers, or
+move from an App Services connection to an Edge Server connection.
+
+Currently, this feature is implemented as an experimental module that requires
+specific TypeScript configuration. The required configuration can be difficult
+to make work with a React Native app. Refer to :ref:`documentation for the
+Node.js SDK <node-change-server-typescript>` for more information.
```
Comment (lindseymoore) [inline source/sdk/react-native/app-services/connect-to-app-services-app.txt:124]:
```suggestion
to make work with a React Native app. For more information on implementation, refer to :ref:`documentation for the
Node.js SDK <node-change-server-typescript>`.
```

**Review by lindseymoore — APPROVED** [review]
Small comment, otherwise LGTM!

File: source/sdk/react-native/app-services/connect-to-app-services-app.txt (hunk 1 above)
Comment (lindseymoore) [reply source/sdk/react-native/app-services/connect-to-app-services-app.txt:97]: A shame :(

**Pull request merged**